src/Stripe.Tests.XUnit/BaseStripeTest.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Linq;
+using System.Net.Http;
+using Xunit;
+
+namespace Stripe.Tests.Xunit
+{
+    public class BaseStripeTest : IDisposable
+    {
+        /// <value>Minimum required version of stripe-mock</value>
+        /// <remarks>
+        /// If you bump this, please don't forget to bump STRIPE_MOCK_VERSION in appveyor.yml.
+        /// </remarks>
+        private const string MockMinimumVersion = "0.19.0";
+
+        private string port;
+
+        private string OrigApiBase;
+        private string OrigUploadBase;
+        private string OrigApiKey;
+
+        public BaseStripeTest()
+        {
+            CheckStripeMock();
+            SetUpStripeMock();
+        }
+
```

Comment:
One thing I liked in @remi-stripe's other PR was a destructor. Having additional cleanup area could prove useful. 

~BaseStripeTest()
{
}